_posts/en/posts/2025-10-24-disclose-cve-2025-46597.md

```diff
@@ -0,0 +1,55 @@
+---
+title: CVE-2025-46597 - Highly unlikely remote crash on 32-bit systems
+name: blog-disclose-cve-2025-46597
+id: blog-disclose-cve-2025-46597
+lang: en
+type: advisory
+layout: post
+redirect_from:
+  - /en/2025/10/24/disclose-cve-2025-46597
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement. English posts only
+announcement: 1
+
+excerpt: >
+  An attacker could produce a block that crashes nodes running on 32-bit systems in a rare edge case. A fix was released on October 10th 2025 in Bitcoin Core v30.0.
+---
+
+Disclosure of the details of a bug on 32-bit systems which may, in a rare edge case, cause the node
+to crash when receiving a pathological block. This bug would be extremely hard to exploit. A fix was
+released on October 10th 2025 in Bitcoin Core v30.0.
+
+This issue is considered **Low** severity.
+
+## Details
+
+Before writing a block to disk, Bitcoin Core sanity checks its block size. This check would overflow
```

Comment:
The term "sanity check" is not very descriptive, and isn't commonly used this way outside of software development AFAIK
```suggestion
Before writing a block to disk, Bitcoin Core checks that its size is within a normal range. This check would overflow
```